src/poly/domain.rs

```diff
@@ -268,6 +252,102 @@ impl<G: Group> EvaluationDomain<G> {
         }
     }
 
+    fn op_extended(
+        &self,
+        mut left: Polynomial<G, ExtendedLagrangeCoeff>,
+        right: &Polynomial<G, ExtendedLagrangeCoeff>,
+        rotation: Rotation,
+        op: impl Fn(&mut G, &G) + Send + Sync + 'static,
+    ) -> Polynomial<G, ExtendedLagrangeCoeff>
+    where
+        G: Field,
+    {
+        // Rotation while in the extended domain is simply exaggerated by how
+        // much larger the extended domain is compared to the original domain.
```

Comment:
I like exaggerated (scaled isn't accurate either I think) because the act of "rotation" is to, well, rotate the evaluations. It is indeed an "exaggerated" rotation because being in the extended domain means that we have to rotate more and it's proportional to how much larger the domain is.